main.go

```diff
@@ -59,6 +60,7 @@ func main() {
 		gcpClusterConcurrency int
 		gcpMachineConcurrency int
 		syncPeriod            time.Duration
+		healthAddr            string
```

Comment:
I think you need to pass this into the controller-manager Options via HealthProbeBindAddress

Reply:
good catch, forgot that, thanks!